CHANGELOG.md

```diff
@@ -5,17 +5,16 @@ Appium 1.9.1 is a patch release
 
 #### iOS
 * Support Xcode 10 and iOS 12.0
-* Fix container type parsing
+* Add a possibility to set container type for Simulator
 * Fix W3C format handling by receiveAsyncResponse command
 
 #### Android
 * Fix `getStrings` for no app
 * Return the current geolocation from `setGeoLocation`
-* Add /location to the list of NO_PROXY endpoints (UiAutomator2 only)
+* Fix handling of getting geolocation endpoint (UiAutomator2 only)
```

Comment:
this is not important and related to `Return the current geolocation from ` point

Reply:
Really? That commit (https://github.com/appium/appium-uiautomator2-driver/commit/c69adf7b6cceb591e181bdd4fa60abb2f2838c91) adds `GET` requests for `location` to the list. That seems different than setting the location.

Reply:
Feel free to keep it if you like. For my taste it's a minor change